README.md

```diff
@@ -36,16 +36,15 @@ npm install
 
 There are a few options to change the behaviour of `hoodie-server`.
 
-option        | default                           |   description
-------------- | --------------------------------- | -------------
-path          | process.cwd()                     | Project path
```

Comment:
It just adds the functionality that you can run hoodie-server outside your hoodie app folder. e.g.: `hoodie-server --path /my/hoodie/app`